tests/integration/test_dbgap.py

```diff
@@ -0,0 +1,315 @@
+"""
+This test file tests annotations by using multiple NER tools against a source
+of data that is of immediate use to [Dug](https://github.com/helxplatform/dug):
+data dictionaries from dbGaP. The code to do this is closely related to that
+used by [`bin/get_dbgap_data_dicts.py`](https://github.com/helxplatform/dug/blob/4741844d51e18cd59a64f7537aa515690239da49/bin/get_dbgap_data_dicts.py),
+except that this code is simpler: we only download dbGaP dictionaries of
+interest directly.
+
+As part of this test, we write two files to tests/integration/data/test_dbgap:
+- The data dict filename (e.g. `phs000810.v1.pht004715.v1.HCHS_SOL_Cohort_Subject_Phenotypes.data_dict.xml`).
+- A custom JSON format which consists of a list of PubAnnotation entries, one for each field in the data dictionary
+  and tracks for each methodology that we are using, with a separate track for normalized identifiers where possible.
+
+This is an integration test: it will use two API endpoints that can be configured
+by environmental variables:
+- The Nemo-Serve URL, which defaults to https://med-nemo.apps.renci.org/
+- The SAPBERT URL, which defaults to https://med-nemo-sapbert.apps.renci.org/
+
+You can run all tests in this file by running `pytest tests/integration/test_dbgap.py` or running individual
+test in this file by using -k command line option to select tests to run based on their name,
+for example, `pytest tests/integration/test_dbgap.py -k 'test_download_dbgap_data_dict'`.
+"""
+
+import json
+import logging
+import os
+import urllib.parse
+import xml.etree.ElementTree as ET
+import requests
+import pytest
+
+# Configuration: get the Nemo-Serve URL and figure out the annotate path.
+NEMOSERVE_URL = os.getenv('NEMOSERVE_URL', 'https://med-nemo.apps.renci.org/')
+NEMOSERVE_ANNOTATE_ENDPOINT = urllib.parse.urljoin(NEMOSERVE_URL, '/annotate/')
+NEMOSERVE_MODEL_NAME = "token_classification"
+
+# Configuration: get the SAPBERT URL and figure out the annotate path.
+SAPBERT_URL = os.getenv('SAPBERT_URL', 'https://babel-sapbert.apps.renci.org/')
+SAPBERT_ANNOTATE_ENDPOINT = urllib.parse.urljoin(SAPBERT_URL, '/annotate/')
+SAPBERT_MODEL_NAME = "sapbert"
+
+# Configuration: the `/get_normalized_nodes` endpoint on a Node Normalization instance to use.
+NODE_NORM_ENDPOINT = os.getenv('NODE_NORM_ENDPOINT', 'https://nodenormalization-sri.renci.org/1.3/get_normalized_nodes')
```

Comment:
can this just point to `https://nodenormalization-sri.renci.org/get_normalized_nodes` with out 1.3 path

Reply:
D'oh! Yes, of course. Fixed in 6c167fae7076ff8a128fc256c88902e4928fdebc.